main.go

```diff
@@ -1,11 +1,17 @@
 package main
 
 import (
+	"log"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud"
 )
 
 func main() {
+	// Remove any date and time prefix in log package function output to
+	// prevent duplicate timestamp and incorrect log level setting
+	log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
```

Comment:
can the **timestamp=...** be deleted in log?

Reply:
this line can remove the timestamp added by log package, but the timestamp=... is printed by TF, so we can't remove it